webui/src/lib/components/repository/ObjectsDiff.jsx

```diff
@@ -101,13 +139,41 @@ const ContentDiff = ({config, repoId, path, leftRef, rightRef, leftSize, rightSi
             oldValue={left?.response}
             newValue={right?.response}
             splitView={false}
-            useDarkTheme={state.settings.darkMode}
+            useDarkTheme={settings.darkMode}
             compareMethod={DiffMethod.WORDS}
         />
 
     </div>;
 }
 
+const ImageDiff = ({oldUrl, newUrl, leftSize, rightSize, diffType}) => (
+    <div>
+        <ImageDiffSummary leftSize={leftSize} rightSize={rightSize} diffType={diffType} />
+        <div style={{ display: 'flex', gap: '1rem' }}>
+            {oldUrl && (
+                <Card style={{ flex: 1 }}>
+                    <Card.Header className="text-danger text-center" style={{ backgroundColor: '#ffeef0', padding: '0.5rem' }}>
```

Comment:
Please avoid adding inline styling, unless there's a very good reason.
Use bootstrap classes, or add classes to the relevant css.